example/example.js

```diff
@@ -62,15 +62,16 @@ shell.on("gl-init", function() {
     contourLevels.push(20*(i+0.3)/6.0)
   }
 
-  
+
   surface = createSurface({
     gl: gl,
     field: field,
     //levels: [contourLevels,contourLevels,contourLevels],
     lineWidth: 3,
     contourTint: 1,
     coords: coords,
-    contourProject: [[true,false,false], [true,false,false], [true,false,false]]
+    contourProject: [[true,false,false], [true,false,false], [true,false,false]],
+    vertexColor: false
```

Comment:
So, if I'm understanding correctly, in plotly.js we would set `vertexColor` to `true` whenever `colorscale[0][0] === colorscale[colorscale.length][0]` in the surface convert [step](https://github.com/plotly/plotly.js/blob/920a333283b1ad9506e804a2e872eaa7f77c3f2e/src/traces/surface/convert.js#L232) ?

cc @rreusser